lib/Pages/otpservice.dart

```diff
@@ -18,20 +18,22 @@ enum UserRole {
 
 class OtpService {
   // Base URL for the API
-  static const String baseUrl = 'https://your-actual-base-url.com/api/v1';
+  static const String baseUrl =
+      'https://ce3e-2409-40e3-1024-c0d1-c840-b228-f755-3df4.ngrok-free.app';
 
   // Method to send OTP for customer login
   Future<Map<String, dynamic>> sendCustomerOtp(String mobileNumber) async {
     try {
       final requestBody = json.encode({'mobileNumber': mobileNumber});
 
       final response = await http.post(
-        Uri.parse('$baseUrl/otp/send-otp'),
+        Uri.parse('$baseUrl/api/v1/otp/send-otp'),
```

Comment:
update base url => '$baseUrl/api/v1/ then add other things

because api/v1 may be change